### ISSUE TITLE
Writing small fees with comma and leading 0 results in deleting first 0 after a comma

### ISSUE THREAD

**Issue by boldninja** [ArkEcosystem/desktop-wallet#1421]
If you try to write fees manually and start with 0,0 (must be comma) it deletes the first zero after a comma so you cannot input smaller fees. A workaround at the moment is to use . (dot) eg. 0.0xxx.

Would be nice to have both versions working as intended since some EU countries use comma as the numeral decimal denominator instead of dot.

**Comment by zillionn** [comment]
I also don't like when I'm changing the fee and if the fee is `0.002` and I want to make it `0.001`, when I click backspace only once, it will change it to `0` not to `0.00`.

**Comment by alexbarnsley** [comment]
Labeled as refactor to improve the current implementation
